PyRDF/backend/Backend.py

```diff
@@ -53,6 +53,12 @@ def __init__(self, config={}):
 
         """
         self.config = config
+        # Users can set a function to initialize the environment
+        self.initialization = None
+
+    def registerInitialization(self, fun, *args, **kwargs):
+        import functools
+        self.initialization = functools.partial(fun, *args, **kwargs)
```

Comment:
So this means the initialization function will be an attribute of the backend. If the user switches backend, they will need to invoke `initialize` again.
I am thinking if we should move this API function to the backend, then.

Reply:
Currently the attribute is assigned to the `Backend` metaclass, hence switching backends should not require users to invoke the `initialize` method again.

The runtime backend (`Local` or `Dist`) will try to run the initialization method on the mapper ([here](https://github.com/JavierCVilla/PyRDF/pull/7/files#diff-e657c12574bba6547cc9fcf27ee09ec7R63)).

Reply:
Ok I see, then good! As a follow up of my comments below, we could then assign a no-op to the initialization function in the constructor of `Backend`.

Reply:
This was not true, since `initialization` was an attribute bound to the instance, not to the class. Last commit fixes this.